packages/wells/src/client/api/wellboresApi.ts

```diff
@@ -40,10 +41,27 @@ export class WellboresAPI {
       name: wellbore.name,
       externalId: wellbore.externalId,
       wellId: wellbore.wellId,
-      trajectory: async (): Promise<Survey | undefined>  => {return await this.surveys.getTrajectory(wellbore.id).then(response => response).catch(err => err)}
+      trajectory: async (): Promise<Survey | undefined>  => {return await this.surveys.getTrajectory(wellbore.id).then(response => response).catch(err => err)},
+      sourceAssets: async (source?:string): Promise<Asset[] | undefined>  => await this.getSources(wellbore.id, source)
     };
   }
 
+  private getSources = async (wellboreId: Number, source?: string): Promise<Asset[] | undefined> => {
```

Comment:
`wellboreId: Number` should be `wellboreId: number`. (number, string, etc) are primitive types and (Number, String ,Boolean, etc)  are object types